### ISSUE TITLE
quest 12686:  servant of drakuru can't attack

### ISSUE THREAD

**Issue by SimonChenyongxing** [TrinityCore/TrinityCore#26829]
### Description

After use the scepter of empowerment of a servant of drakuru,  he don't attack darmuk.
but the follow and stand command works.

### Expected behaviour

When I select Darmuk, and click attack button, he should attack Darmul.

### Steps to reproduce the problem

follow the quest guide.

### Branch

3.3.5

### TC rev. hash/commit

feba07fa1e

### Operating system

Windows 7 x64

### Custom changes

no

**Comment by Aokromes** [comment]
last seen completed
12686	186469	2021-06-04 00:13:09	2021-06-04 00:31:46		0	e7e47866066f+	2021-05-31 20:35:03 +0300

**Comment by dr-j** [comment]
Yes ofc quest works if you kill npc the without using the scepter to charm npcs but can confirm charmed creature wont attack darmuk will attack other npcs it should not though, could possibly be due to the force reaction from the scourge disguise and charmed creature but idk

**Comment by SimonChenyongxing** [comment]
Is there plan to fix this?

**Comment by dr-j** [comment]
We need to know exactly why won’t attack first and I’m sure I previously tried fixing this without success suspect will work if faction changed but that would be hack.

**Comment by Thor1423** [comment]
> We need to know exactly why won’t attack first and I’m sure I previously tried fixing this without success suspect will work if faction changed but that would be hack.
@dr-j  @Aokromes 

with the c++ script it works 

https://github.com/TrinityCore/TrinityCore/issues/30286#issuecomment-3155914134
